crates/rpc-types-trace/src/parity.rs

```diff
@@ -821,4 +822,31 @@ mod tests {
         let serialized = serde_json::to_string_pretty(&trace).unwrap();
         similar_asserts::assert_eq!(serialized, reference_data);
     }
+    #[test]
+    fn test_nethermind_trace_result_null_output_value() {
+        let reference_data = r#"{
+  "output": null,
+  "stateDiff": {
+    "0x5e1d1eb61e1164d5a50b28c575da73a29595dff7": {
+      "balance": "=",
+      "code": "=",
+      "nonce": "=",
+      "storage": {
+        "0x0000000000000000000000000000000000000000000000000000000000000005": {
+          "*": {
+            "from": "0x0000000000000000000000000000000000000000000000000000000000042f66",
+            "to": "0x0000000000000000000000000000000000000000000000000000000000042f67"
+          }
+        }
+      }
+    }
+  },
+  "trace": [],
+  "vmTrace": null,
+  "transactionHash": "0xe56a5e7455c45b1842b35dbcab9d024b21870ee59820525091e183b573b4f9eb"
```

Comment:
is this a real transaction?
wasn't able to find it

Reply:
https://gnosisscan.io/tx/0xe56a5e7455c45b1842b35dbcab9d024b21870ee59820525091e183b573b4f9eb

I reduced it slightly